gui_mac.spec

```diff
@@ -39,3 +53,10 @@ app = BUNDLE(exe,
               'CFBundleShortVersionString': __version__
              }
        )
+
+print("Creating zip")
+os.chdir("./dist/")
+zipf = zipfile.ZipFile('./Circleguard_osx.app.zip', 'w', zipfile.ZIP_DEFLATED)
+zipdir('./Circleguard.app/', zipf)
+zipf.close()
+print("Finished zip")
```

Comment:
This gets zipped up into a `Contents` folder, not a `.app`. I think you've gone one level too deep? Every macos application has a `Contents` folder directly inside the `.app`.
![image](https://user-images.githubusercontent.com/31628143/81425013-cea52100-9124-11ea-9f0b-890deca56c4b.png)